docs/content/Getting-Started/Cloud/02-Create-a-deployment.mdx

```diff
@@ -0,0 +1,104 @@
+---
+title: Create a deployment
+permalink: /getting-started/cloud/create-a-deployment
+category: Getting Started
+subCategory: Cube Cloud
+menuOrder: 3
+isDisableFeedbackBlock: true
+---
+
+## Create deployment
+
+The first step in creating a deployment is to sign in to your Cube Cloud
+account. After signing in, click the **Create Deployment** button:
+
+<div style="text-align: center">
+  <img
+    alt="Cube Cloud Landing Screen with no configured deployments"
+    src="https://ucarecdn.com/f3ec1ae0-1d5d-4b5b-9859-9fc145b38e43/"
+    style="border: none"
+    width="80%"
+  />
+</div>
+
+Give the deployment a name and select the cloud provider and region of your
+choice and click **Next**:
+
+<div style="text-align: center">
+  <img
+    alt="Cube Cloud Create Deployment Screen"
+    src="https://ucarecdn.com/2338323e-0db8-4224-8e7a-3b4daf9c60ec/"
+    style="border: none"
+    width="80%"
+  />
+</div>
+
+## Set up Cube project
+
+Next, click **Create** to create a new project from scratch:
+
+<div style="text-align: center">
+  <img
+    alt="Cube Cloud Upload Project Screen"
+    src="https://ucarecdn.com/46b72b61-b650-4271-808d-55203f1c8d8b/"
+    style="border: none"
+    width="80%"
+  />
+</div>
+
+## Connect a data source
+
+The last step in creating a deployment is to connect your data source to Cube
+Cloud.
+
+<InfoBox>
+
+Want to use a sample database instead? Use the credentials below:
+
+<br />
+
+| Field    | Value              |
+| -------- | ------------------ |
+| Host     | `demo-db.cube.dev` |
+| Port     | `5432`             |
+| Database | `ecom`             |
+| Username | `cube`             |
+| Password | `12345`            |
+
+</InfoBox>
+
+To do this, first select the data source:
```

Comment:
In the info box above, it's not specified that the sample database is Postgres. So, it's unclear what should one select here. Only later the image says "Postgres."